README.md

```diff
@@ -1,2 +1,10 @@
 # github-connector
-Hello!
\ No newline at end of file
+## How to run project with Docker
+### **[Docker Required](https://www.docker.com/)**
```

Comment:
We might want to recommend installing Docker Desktop. For some versions of Docker, you need to install docker-compose separately: https://docs.docker.com/compose/install/

Reply:
Updated README to point to your link

Reply:
As a troubleshooting step, if you see `command not found` when running `docker-compose up`, you probably need to make sure the docker service is running